setuptools_odoo/make_default_setup.py

```diff
@@ -120,6 +121,62 @@ def make_default_setup_addons_dir(addons_dir, force,
                                  odoo_version_override)
 
 
+def clean_setup_addons_dir(addons_dir):
+    paths_to_remove = []
+
+    addons_setup_dir = os.path.join(addons_dir, 'setup')
+    for addon_name in os.listdir(addons_setup_dir):
+        addon_setup_dir = os.path.join(addons_setup_dir, addon_name)
+        is_directory = os.path.isdir(addon_setup_dir)
+        skip_entry = not is_directory or addon_name == '_metapackage'
+        if skip_entry:
+            continue
+
+        addon_dir = os.path.join(addons_dir, addon_name)
+        is_installable = is_installable_addon(addon_dir)
+        try:
+            # File or directory is not an addon. Skip it
+            manifest = read_manifest(addon_dir)
+        except NoManifestFound:
+            manifest = None
+
+        if not is_installable or manifest is None:
```

Comment:
s/manifest is None/not manifest/

In general in python it's better to test for falsiness than behing too specific. It's easier to read and more robust.